### ISSUE TITLE
Shared object bit count mismatch with python interpreter error.

### ISSUE THREAD

**Issue by CaiusTSM** [overdev/raylib-py#18]
I get the following error when importing raylibpy:

```
import raylibpy
  File "/home/----/.local/lib/python3.8/site-packages/raylibpy/__init__.py", line 71, in <module>
    _rl = CDLL(os.path.join(RAYLIB_BIN_PATH, _lib_filename[_platform]))
  File "/usr/lib/python3.8/ctypes/__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/----/.local/lib/python3.8/site-packages/raylibpy/libraylib.so.2.0.0: wrong ELF class: ELFCLASS32
```

I believe this is due to running 64bit python, but the shared object included in this project is 32bit. Could you also include a 64bit version that is dlopened instead when running 64bit python?

**Comment by Tau5** [comment]
I second this

**Comment by overdev** [comment]
Fixed in 4.2.0.post2
